src/integrations/magni.ts

```diff
@@ -0,0 +1,16 @@
+import fetch, { Response } from 'node-fetch';
+import { fetchOptions } from '../http';
+import { FastifyRequest } from 'fastify';
+
+const magniOrigin = process.env.MAGNI_ORIGIN;
+
+export const postFeedback = async (req: FastifyRequest): Promise<Response> =>
+  fetch(`${magniOrigin}/feedback`, {
+    ...fetchOptions,
+    method: 'post',
+    body: JSON.stringify(req.params),
+    headers: {
+      cookie: req.headers.cookie,
```

Comment:
This is not required, instead you have to provide `X-User-Id` header as mentioned in the DR.
<img width="294" alt="image" src="https://github.com/dailydotdev/daily-api/assets/1993245/d243dd87-2333-45be-ac32-e64cac0cf4a3">

Reply:
Right. Let me fix this quickly.